core/backend/src/IModelDb.ts

```diff
@@ -1033,6 +1033,14 @@ export abstract class IModelDb extends IModel {
       metaData.baseClasses.forEach((baseClassName: string) => this.loadMetaData(baseClassName));
   }
 
+  /** Returns the full schema for the input name or undefined if the schema does not exist in this iModel.
+   * @param name The name of the schema e.g. 'BisCore'
+   * @returns The SchemaProps for the requested schema or undefined
+   */
+  public getSchemaProps(name: string): IModelJsNative.SchemaProps {
```

Comment:
I am a bit concerned here returns IModelJsNative.SchemaProps from a public method.  I have seen examples of reexporting as a type in other cases.  e.g. some CloudSQLite types: https://github.com/iTwin/itwinjs-core/blob/b55e4ecff9423b54cfdfe65da159747438cd0195/core/backend/src/SQLiteDb.ts#L240

Reply:
Yes, the documentation generator will complain. Probably you should move SchemaProps into core-common (i eventually will have to do that with the CloudSqlite stuff too, but it's all internal now).

Reply:
@ColinKerr would you like me to finish this PR?